x11-misc/gpaste/gpaste-3.36.3.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+VALA_MIN_API_VERSION="0.30"
+VALA_USE_DEPEND="vapigen"
+
+inherit meson vala gnome2-utils
+
+DESCRIPTION="Clipboard management system"
+HOMEPAGE="https://github.com/Keruspe/GPaste"
+SRC_URI="https://github.com/Keruspe/GPaste/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+introspection +gnome vala"
+REQUIRED_USE="
+	vala? ( introspection )
+	gnome? ( introspection )
+"
+
+DEPEND="
+	dev-libs/appstream-glib
+	>=dev-libs/glib-2.48:2
+	introspection? (
+		>=dev-libs/gobject-introspection-1.48.0
+	)
+	sys-apps/dbus
+	>=x11-libs/gdk-pixbuf-2.34:2
+	>=x11-libs/gtk+-3.20:3
+	x11-libs/libX11
+	x11-libs/libXi
+	gnome? (
+		media-libs/clutter
+		x11-libs/pango
+	)
+"
+BDEPEND="
+	vala? ( $(vala_depend) )
+	gnome? (
+		gnome-base/gnome-control-center:2
+	)
+	virtual/pkgconfig
+"
+RDEPEND="${DEPEND}
+	gnome? (
+		gnome-base/gnome-control-center:2
+		gnome-base/gnome-shell
+	)
+"
+
+S=${WORKDIR}/GPaste-${PV}
+
+src_prepare() {
+	use vala && vala_src_prepare
+	default
+}
+
+src_configure() {
+	local emesonargs=(
+		-Dsystemd=true
```

Comment:
I am sorry to return this to you... but:
```
Run-time dependency systemd found: NO (tried pkgconfig and cmake)

meson.build:47:4: ERROR: Dependency "systemd" not found, tried pkgconfig and cmake

A full log can be found at /var/tmp/portage/x11-misc/gpaste-3.36.3/work/gpaste-3.36.3-build/meson-logs/meson-log.txt
 * ERROR: x11-misc/gpaste-3.36.3::testworld failed (configure phase):
```
So it does seem to be linking against libsystemd.so. I warned you about it ;)
You'll have to make it an USE flag here. Remember to modify DEPEND+RDEPEND correctly.

Well at least we seem to have moved on from gnome-keybindings!

I warned you it'd be a huge task, but as you might figure out, we don't want to push a broken ebuild to the tree. If we can't compile it, chances are so wouldn't other people.

Reply:
All good. My machine and test VM are systemd so did not pick that up. I'll setup a non-systemd machine to verify my changes and get back